bitrise.yml

```diff
@@ -0,0 +1,91 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test:
+    before_run:
+    - audit-this-step
+    steps:
+    - go-list:
+    - golint:
+    - errcheck:
+    - go-test:
+    - change-workdir:
+        title: Switch working dir to ./_tmp dir
+        inputs:
+        - path: ./_tmp
+        - is_create_path: true
+    - path::./:
+        title: Step Test
+        inputs:
+        - new_version: 1.0.$BITRISE_BUILD_NUMBER
+    - script:
+        title: Output test
+        inputs:
+        - content: |-
+            echo '$BITRISE_CHANGELOG='
+            echo "$BITRISE_CHANGELOG"
+            if [[ -z $BITRISE_CHANGELOG ]] ; then
+              echo 'empty $BITRISE_CHANGELOG'
+              exit 1
+            fi
+
+  audit-this-step:
+    steps:
+    - script:
+        title: Audit step.yml
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            stepman audit --step-yml ./step.yml
+
+  release:
```

Comment:
i think for now we should use the 'old' release wf.
the listed steps are not yet released (also the step ids are not correct): `steps-generate-changelog`, `steps-github-release`